### PR TITLE
Fetcher proto

### PR DESCRIPTION
**Pull request by alexander871015**
Update proto such that the enums are alphabetical for easier readability

### CODE REVIEW

File: hologram_dashboard/fetcher/proto/hologram_availability.proto [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+message HologramDataAvailability {
+  // Specification about which data source is currently accessed
+  optional DataType data_source = 1;
+
+  // Specification about the status of the data source
+  message DataSourceDetail {
+      optional int64 date = 1;
+      optional StatusType status = 2;
+  }
+
```
Comment (emilyqu) [inline hologram_dashboard/fetcher/proto/hologram_availability.proto:30]: nit: move the DataSourceDetail out?

message DataSourceDetail {
}

message HologramDataAvailability {
}

File: hologram_dashboard/fetcher/hologram_availability_fetcher.h [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+        */
+        void sendToDatabase();
+
+        /* 
+        * Requires: path is a valid path to a config file
+        * Modifies: hologram_configs
+        * Effects: populate information on configs of the hologram
+        */
+        void populateConfigs(std::string path);
+
```
Comment (emilyqu) [inline hologram_dashboard/fetcher/hologram_availability_fetcher.h:93]: Simplify apis in this file as discussed offline?

**Comment by alexander871015** [comment]
@emilyqu this one contains only the protos

File: hologram_dashboard/fetcher/hologram_availability_fetcher.h (hunk 2 above)
Comment (alexander871015) [reply hologram_dashboard/fetcher/hologram_availability_fetcher.h:93]: this is changed at a separate pr

File: hologram_dashboard/fetcher/proto/hologram_availability.proto (hunk 1 above)
Comment (alexander871015) [reply hologram_dashboard/fetcher/proto/hologram_availability.proto:29]: Done

**Pull request closed without merging**